### PR TITLE
[Admin] fix edit and create space preview

### PR DESCRIPTION
**Pull request by denisdimitrov**
Fixes the space edit and create preview and uses the correct image for background in admin dashboard.

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1354

### CODE REVIEW

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Few notes about refactor and an issue about the preview not updating without clicking save (unlike the world preview).

sparkle-space-preview-01


sparkle-world-preview-01

File: src/hooks/useCurrentSpace.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  isLoaded: boolean;
+  space?: WithId<Space>;
+}
+
+/**
+ * Hook that returns the space when an id is provided.
+ * @param param0 This is the id of the space that is fetched from the collection.
+ * @returns Object with 'space' and 'isLoaded'
+ */
+export const useCurrentSpace: ReactHook<
```
[the remark this answers is not in the document]
Comment (denisdimitrov) [reply src/hooks/useCurrentSpace.ts:24]: @goran-peoski-work The other hooks are using the old useConnect syntax and it's 'lagging' just like the world did. It's showing the previous venue data when opened. That's why I decided to add this.

File: src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx [hunk 2]
```diff
@@ -31,7 +34,7 @@ const VenueWizardCreate: React.FC<VenueWizardCreateProps> = () => {
 
   return (
     <AdminRestricted>
-      <Details />
+      <Details state={state} dispatch={dispatch} />
```
[the remark this answers is not in the document]
Comment (denisdimitrov) [reply src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:37]: I just copied the old code, it's not good or very optimized.

[the remark this answers is not in the document]
Comment (goran-peoski-work) [reply src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:37]: What old code?

File: src/hooks/useCurrentSpace.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (goran-peoski-work) [reply src/hooks/useCurrentSpace.ts:24]: which comes to
👇 
> should they get a @deprecated / @debt comments?

**Comment by whoaitsaimz** [comment]
This one will be outdated as soon as Alexey's PR gets reviewed & merged sparkletown/sparkle#2533

File: src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (denisdimitrov) [reply src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx:37]: It's somewhere in the old PR's its really hard to find it, but this was basically the old implementation that was removed by me. I will improve this as much as possible.

**Comment by ermiaSparkle** [comment]
@denisdimitrov Should we close this one ?

**Comment by denisdimitrov** [comment]
The new design change makes this obsolete.

**Pull request closed without merging**